controllers/users.js

```diff
@@ -5,14 +5,23 @@ const UsersController = {
     res.render("users/new", {});
   },
 
-  Create: (req, res) => {
+  Create: async (req, res) => {
     const user = new User(req.body);
-    user.save((err) => {
-      if (err) {
-        throw err;
-      }
-      res.status(201).redirect("/posts");
-    });
+
+    const userExists = await User.exists({email: user.email})
+
+    if (userExists == false) {
+      user.save((err) => {
+        if (err) {
+          throw err;
+        }
+        res.status(201).redirect("/posts");
```

Comment:
there is a test for 'user signs up and is redirected to sign in'.  Here it looks like when the user signs up they are redirected to posts?

Reply:
Hey Harry that branch of the if statement is for when the user fails to sign up i.e. the users emails is already registered.

However, should line 18 redirect to sessions/new